Dockerfile

```diff
@@ -134,22 +134,19 @@ RUN apt remove -y unzip
 # CVE-2019-20916, PRISMA-2022-0168
 RUN rm -rf /usr/lib/python3.6
 
-## Default user
-#USER <default user>:0
-USER ${USER_UID}:${USER_GID}
```

Comment:
From a security perspective the image should not be executed as `root`.  

See: 
https://stackoverflow.com/questions/19054029/security-of-docker-as-it-runs-as-root-user
https://medium.com/@amandubey_6607/container-security-best-practices-avoid-running-containers-as-root-ae37fee2846f

Reply:
I agree that running the entire container as non-root would be preferred, but AFAICS then there is no way to automatically handle the permission setup. Doing that manually is possible, but more cumbersome and raises the barrier for setting this up quickly (I was also worried that you cannot easily do a manual `chown` on named volumes which are somewhat hidden in `/var/lib/docker/volumes`, but I just tested that docker actually copies the permissions of the directory inside the container to the named volume outside of the container when the volume is created - pity this does not happen for mounted directories when they are created by docker...).

Anyway, also note that with my commit, the application is still not ran as root - the `setpriv` in `entrypoint.sh` drops root privileges after the `chown`, so the application still runs as before, so there should not be significant extra attack surface compared to before (see also https://stackoverflow.com/a/65576874/740048 which analyzes exactly this case).

But it's your call. If you prefer manually setting the permissions, I can drop the commit (and replace it with improving docs on manually setting the permissions). If I've convinced you that the current approach is acceptable in terms of security, then I think no changes are needed.

Reply:
Considering this is available to the community, if anyone wants to harden the container/image they can undertake specific steps to do so.  For the general user your changes will improve accessability.